### PR TITLE
[ADD] added module web_graph_radar

### PR DESCRIPTION
**Pull request by veryberry**

### CODE REVIEW

File: web_graph_radar/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Graph Radar Chart.
+=================
+
+Add new graph view mode: Radar
```
Comment (yelizariev) [inline web_graph_radar/README.md]: Use this template for readme
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

Comment (yelizariev) [inline web_graph_radar/README.md]: Then add there to Contributors yourself and 

```
 Florent THOMAS (financial support) <https://github.com/MindAndGo>
```

File: web_graph_radar/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Radar Chart',
+    'version': '8.0.0.1.0',
+    'category': 'Web',
+    'summary': 'Add graph radar view.',
+    'author': "Veronika Kotovich,Odoo Community Association (OCA)",
```
Comment (yelizariev) [inline web_graph_radar/__openerp__.py]: Could you please add the company as well

```
'author': "Veronika Kotovich,IT-Projects LLC,Odoo Community Association (OCA)",
```

File: web_graph_radar/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline web_graph_radar/__openerp__.py:1]: Please simplify headers following https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

File: web_graph_radar/README.rst [hunk 4]
```diff
@@ -0,0 +1,45 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
```
Comment (elicoidal) [inline web_graph_radar/README.rst]: Please adjust the lines to the length of the title

File: web_graph_radar/README.rst [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+Graph Radar Chart
+==============
+
+This module extends the functionality of web_graph module to support radar mode
```
Comment (elicoidal) [inline web_graph_radar/README.rst:9]: period

File: web_graph_radar/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Radar Chart',
+    'version': '8.0.0.1.0',
+    'category': 'Web',
+    'summary': 'Add graph radar view.',
+    'author': "Veronika Kotovich,IT-Projects LLC,Odoo Community Association (OCA)",
```
Comment (elicoidal) [inline web_graph_radar/__openerp__.py]: Actually usually  the individuals are listed in contributors and company in authors

**Comment by elicoidal** [comment]
Thanks for the contribution! You might want to propose it to Odoo master

**Comment by yelizariev** [comment]
@elicoidal I believe it will not be accepted in odoo master for the same reason why they removed Radar view from v8 (I don't know those reasons)

File: web_graph_radar/__openerp__.py [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Radar Chart',
+    'version': '8.0.0.1.0',
```
Comment (yelizariev) [inline web_graph_radar/__openerp__.py]: 8.0.1.0.0

**Comment by elicoidal** [comment]
OK: I dont know either indeed.

**Comment by veryberry** [comment]
@yelizariev they removed radar because they moved to another graphic lib d3.js+nvd3.js for drawing charts, and these libs don't support radar by default.

**Comment by flotho** [comment]
Hi @veryberry , @yelizariev ,

It looks like it causes the crash of all the interface
odoo_v10

Tested here : http://3143992-328-1eac03.runbot2.odoo-community.org/web#view_type=graph&model=account.invoice&menu_id=235&action=273

**Comment by veryberry** [comment]
@flotho I think it's module web_menu_autohide in action, web_graph_radar does not affect interface stylesheet.

**Comment by yelizariev** [comment]
@flotho OCA's runbot installs all modules from this repository. It could be better to install only updated ones. Here the solution for this if someone is interested: https://github.com/yelizariev/runbot-addons/blob/8.0/runbot_custom/models.py#L54-L63

**Comment by lmignon** [comment]
Installing all modules help to detect and prevent incompatibility between
modules...
Le 20 mars 2016 07:48, "Ivan Yelizariev" notifications@github.com a
écrit :

**Comment by flotho** [comment]
@veryberry , thanks, thanks to you I also discovered a new module.

regards

**Comment by nono6677** [comment]
Hi @veryberry , @yelizariev ,
I am a colleague of @flotho.

Is it possible to add a .png export ?

regards

**Comment by yelizariev** [comment]
@nono6677 it's no so easy I guess

**Comment by nono6677** [comment]
Ok.
Otherwise, is it possible to set the max value ( Scale of graph)

**Comment by flotho** [comment]
Hi @yelizariev ,

The library in v7 is supposed to support png export :
odoo_v10

Maybe an action will be easy to implement.

What do you think about this?

**Comment by yelizariev** [comment]
@flotho it should help
But such update is not related to radar view and could be applied to other graph views too

**Comment by flotho** [comment]
Ok, thanks for your answer, we'll try to find when Odoo delete this action from BI view

**Comment by yelizariev** [comment]
@veryberry could your "export as png" feature be a separate module, e.g. web_graph_export_png?

**Comment by yelizariev** [comment]
@flotho please review

**Comment by yelizariev** [comment]
@lmignon
I have made updates for runbot, that allow to run two instances: one for updated modules only and one for all
https://twitter.com/yelizariev/status/732841035693461504

**Comment by flotho** [comment]
Hi @yelizariev ,

From my point of view it works as expected.
regards

File: web_graph_radar/static/lib/nvd3-radar.js [hunk 8]
```diff
@@ -0,0 +1,885 @@
+nv.models.radar = function() {
```
Comment (lasley) [inline web_graph_radar/static/lib/nvd3-radar.js:1]: Can you add a comment at the top of this file indicating the `nvd3-radar` version that is being used here? Not a blocker, just incredibly helpful from a long-term maintainability perspective IMO

**Comment by lasley** [comment]
@veryberry - thank you for the submission. I have reviewed `web_graph_radar`, minor comment inline. There are also a few Lint issues in Travis due to some missing newline chars - https://travis-ci.org/OCA/web/jobs/123872661#L424

 Can you please split the `web_graph_export_png` into a separate PR to be reviewed individually? I will make sure to keep an eye out and review that as well.

**Comment by flotho** [comment]
Hi @yelizariev @veryberry ,
We still have an issue #610  thant we didn't succeed to handle.
Any feedback from you would be appreciated.
Regards

**Pull request closed without merging**